plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleSQLDialect.java

```diff
@@ -728,4 +730,16 @@ public String getClobDataType() {
     public boolean needsDefaultDataTypes() {
         return false;
     }
+
+    @NotNull
+    @Override
+    public String getSchemaExistQuery(@NotNull String schemaName) {
+        return "SELECT 1 FROM all_users WHERE USERNAME='" + schemaName + "'";
+    }
+
+    @NotNull
+    @Override
+    public String getCreateSchemaQuery(@NotNull String schemaName) {
+        return "CREATE USER " + schemaName + " IDENTIFIED BY " + UUID.randomUUID();
```

Comment:
This will fail as UUIDs contain hyphens. You'll need to escape the password.

You might want to escape the name as well.